### DISCUSSION TITLE
Possible Ktor integration

### DISCUSSION THREAD

**Discussion by JoonasC** [jobrunr/jobrunr#193] [Ideas]
I noticed that you are creating integrations/extensions for Quarkus and Micronaut. Kotlin also has a framework for creating an asynchronous restful server applications with routes, etc, that is called Ktor. If I understand correctly the Quarkus/MIcronaut integrations automatically create routes to export statistics on JobRun, among other things? Having that with Ktor would be nice. Additionally, would it be possible for this integration to allow wrapping the JobRunr API with Ktor, this would be nice as essentially running two web servers in one process: one for JobRunr's API and one for Ktor seems kind of redundant.

**Comment by rdehuyss** [comment]
I didn't really plan to do so, to be honest. Time is scarce and JobRunr is taking a lot from it 🙂. 

But, I'm definitely open for a PR on that one!
